### DISCUSSION TITLE
Membership Application

### DISCUSSION THREAD

**Discussion by ScriptingShrimp** [kiali/kiali#4752] [Governance]
Hi, my name is Pavel, 
some of you may not know me, but for last 8 moths I'm part of Kiali team. 

I do not contribute directly in Kiali repositories, but you can see my effort to fix Kiali UI regression suite [here](https://github.com/Kiali-QE/kiali-qe-python/pulls?page=1&q=is%3Apr+author%3AScriptingShrimp) and you may also remember me from Cypress PoC PR [here](https://github.com/kiali/kiali-ui/pull/2264).

I would like to become member of contributors group, to be able to review and send PR's mostly related to testing.

**Comment by lucasponce** [comment]
+1 also @ScriptingShrimp has been pretty active in the regression testing repositories that are currently in private servers and he's participating in improving the QE flows to integrate them with the public Github Actions pipeline.

**Comment by aljesusg** [comment]
💯!! +1 to add Pavel !!

**Comment by nrfox** [comment]
Pavel is already a great contributor with his contributions and continued work for the UI testing. Looking at the formal requirements for [becoming a tester](https://github.com/kiali/kiali/blob/master/GOVERNANCE.md#becoming-a-tester), does "testing pull requests for 3 months or more," include automated testing work? Seems like it should.

**Comment by leandroberetta** [comment]
+1, @ScriptingShrimp thanks for your contributions on Kiali's testing. It's true that you are very active improving test to achieve more quality for our users.

**Comment by mattmahoneyrh** [comment]
Yes @ScriptingShrimp should be a Kiali contributor, and in particular will need the ability to maintain UI automation framework and tests.

**Comment by jshaughn** [comment]
+1, @ScriptingShrimp has a good knowledge of the project and has been contributing for several months.

**Comment by ScriptingShrimp** [comment]
Thanks for support.
Here I'm adding link to another PR https://github.com/kiali/kiali-ui/pull/2323, feel free to review it :)
Improving Kiali overall quality is my main focus.

**Comment by israel-hdez** [comment]
Apparently, this is already fulfilled. So, I've created this: https://github.com/kiali/kiali/pull/4995
Next time, at least drop a comment when membership is granted, so that everybody can welcome the new member to the organization.

**Reply by israel-hdez** [reply]
Welcome @ScriptingShrimp! :-)
